decidim-conferences/app/views/decidim/conferences/conferences/show.html.erb

```diff
@@ -14,71 +14,96 @@ edit_link(
 )
 %>
 
-<%= participatory_space_floating_help %>
+<%# NOTE: This page is wrapped within a main tag, in order to
+    include the hero inside the two-column layout.
+    Therefore is mandatory to pass the "main_enabled: false" flag
+    to the layout_two_col for disabling the original main. %>
+<main>
 
-<div class="row">
-  <div class="columns mediumlarge-12">
-    <section class="section">
-      <h3 class="section-heading"><%= t("conferences.show.introduction", scope: "decidim") %></h3>
-      <%= decidim_sanitize_editor_admin translated_attribute(current_participatory_space.short_description) %>
+  <%= content_for :aside do %>
+    <div class="conference__nav-container">
+      <button id="dropdown-trigger" data-component="dropdown" data-target="dropdown-menu" data-auto-close="true">
+        <span><%= t("decidim.searches.filters.jump_to") %></span>
+        <%= icon "arrow-down-s-line" %>
+        <%= icon "arrow-up-s-line" %>
+      </button>
+      <ul id="dropdown-menu" class="conference__nav">
```

Comment:
Here we have the same "jumping" effect on mobile when the page is initially loaded at this position.

Also at the redesign app this menu is open by default on mobile. Is this on purpose?